build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -169,6 +169,13 @@ function updateReporters(config) {
   if (argv.saucelabs) {
     config.reporters.push('saucelabs');
   }
+
+  if (process.env.TRAVIS && process.env.TRAVIS_EVENT_TYPE === 'push') {
```

Comment:
There's a method called `isTravisPushBuild` in the build-system/common/travis.js file, use it :)

Reply:
Done.